gosu-test/src/test/gosu/gw/specContrib/statements/usesStatement/Errant_ExplicitStaticImportsAllowedByUnqualifiedClassNameFromTheSamePackage.gs

```diff
@@ -0,0 +1,15 @@
+package gw.specContrib.statements.usesStatement
```

Comment:
The ones I spot-checked that are *outside* of specContrib actually have comments or something else about them that suggests there are errors in them...just no issuekeys.  So I assume they are there to test something else besides error highlighting.

I will go ahead and merge this, but I think the argument "if there happens to be an error in these..." is weak. If the file is intended to be legal, then failure to compile *should* cause a test break.